portfolio/src/main/webapp/layout.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tony Shen - {% block heading %}{% endblock %}</title>
+        <link rel="stylesheet" href="style.css">
+        <!--Google Font API-->
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto&effect=anaglyph">
+        <script src="script.js"></script>
+    </head>
+    <body style="page-container">
+        <div class="axis-0-grid-container">
+            <div class="header-container">
+
+                <div class="logo-section">
+                    <img class="logo" src="images/profile.jpg"/>
+                </div>
+
+                <div class="font-effect-anaglyph title-section">Tony Shen</div>
+
+                <div class="navigator-section">  
+                    <a class="font-effect-anaglyph header-links" href="">Projects</a>
+                    <a class="font-effect-anaglyph header-links" href="">CV</a>
+                    <a class="font-effect-anaglyph header-links" href="">Vlog</a>
+                    <a class="font-effect-anaglyph header-links" href="gallery.html">Images</a>
+                </div>
+
+            </div>
+            {% block body %} 
```

Comment:
What is the purpose of this page? If it is not needed, please delete.

Reply:
I was attempting to remove repetitive code using jinja HTML templating, but that did quiet work with just HTML, CSS and JavaScript

Reply:
Removed